packages/lodestar/src/network/reqResp.ts

```diff
@@ -148,6 +156,9 @@ export class ReqResp extends (EventEmitter as IReqEventEmitterClass) implements
     const getResponse = this.getResponse;
     return (source: AsyncIterable<Buffer|BufferList>) => {
       return (async function * () {
+        if (method === Method.Metadata) {
+          yield* getResponse(peerId, method, undefined);
+        }
```

Comment:
Maybe we should introduce better way of handling methods without response

Reply:
Definitely agree we should revisit the request/response logic to better handle methods without request / response.
I'd propose we add an issue and tackle this in a separate PR.

Reply:
:+1:

Reply:
https://github.com/ChainSafe/lodestar/issues/808